protocol-units/bridge/contracts/src/AtomicBridgeCounterparty.sol

```diff
@@ -54,10 +54,10 @@ contract AtomicBridgeCounterparty is IAtomicBridgeCounterparty, OwnableUpgradeab
         address recipient,
         uint256 amount
     ) external onlyOwner returns (bool) {
+        console.log("lockBridgeTransfer start amount:%d", amount);
         if (amount == 0) revert ZeroAmount();
-
         if (atomicBridgeInitiator.poolBalance() < amount) revert InsufficientWethBalance();
-        
+        console.log("verif done");
```

Comment:
Please remove all console.log()s. They only serve for debugging and will weight on tx cost.